test/ui-test/testSuites/suite_messaging/tst_chatsNavigation/test.feature

```diff
@@ -0,0 +1,24 @@
+Feature: Status Desktop Chat Navigation
+
```

Comment:
It would be good to add as a comment, the steps that are done in `onFeatureStart`. See `tst_chatFlow` feature file as a reference.